src/Components/Contact/Contact.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import classes from "./Contact.module.css";
+import {
+  AiFillGithub,
+  AiFillLinkedin,
+  AiFillTwitterCircle,
+} from "react-icons/ai";
+
+const Contact = () => {
+  return (
+    <section id="contact">
+      <div className={classes.container}>
+        <div className={classes.heading}>
+          <h3>Connect with me!</h3>
+        </div>
+        <form>
+          <input type="hidden" name="_subject" value="New submission!" />
+          <div className={classes.input}>
+            <label htmlFor="name">Name</label>
+            <input type="text" required className={classes.singleLine}></input>
+          </div>
+          <div className={classes.input}>
+            <label htmlFor="email">Email</label>
+            <input type="email" required className={classes.singleLine}></input>
+          </div>
+          <div className={classes.input}>
+            <label htmlFor="message">Message</label>
+            <textarea rows="5" col="30" name="message" required></textarea>
+          </div>
```

Comment:
You must give your inputs an ID that match the value you give to your htmlFor attribute on the label. See examples here: https://www.w3schools.com/tags/tag_label.asp